modules/savedb.py

```diff
@@ -22,41 +21,30 @@ def saveToDatabase(database, user, password, links):
 
         #Debug
         #print("Database:", database, "\nuser:",user, "\npass:",password)
-        db = MySQLdb.connect(host="localhost",    # your host
-                                user=user,         # your username
-                                passwd=password,  # your password
-                                db=database)        # name of the data base
-
-        cur = db.cursor()
-
         try:
-                query = """ CREATE TABLE IF NOT EXISTS `tor_url` (
-                id INT(6) UNSIGNED AUTO_INCREMENT PRIMARY KEY,
-                link VARCHAR(30) NOT NULL UNIQUE,
-                reg_date TIMESTAMP)"""
-                cur.execute(query)
-                for link in links:
-                        query = "INSERT IGNORE INTO `tor_url` (link) VALUES ('{0}')".format(link)
-                        cur.execute(query)
-                        #print(query)            
-                db.commit()
-                
-        except (MySQLdb.Error, MySQLdb.Warning) as e:
-                print(e)
-                return None
-
-        except TypeError as e:
-                print(e)
-                return None
-
-        except ValueError as e:
-                print(e)
-                return None
-
-        finally:
-                cur.close()
-                db.close()
-
-
-
-    
\ No newline at end of file
+            db = MySQLdb.connect(host="localhost1",    # your host
+                                    user=user,         # your username
+                                    passwd=password,  # your password
+                                    db=database)        # name of the data base
+
+            cur = db.cursor()
+            try:
+                    query = """ CREATE TABLE IF NOT EXISTS `tor_url` (
+                    id INT(6) UNSIGNED AUTO_INCREMENT PRIMARY KEY,
+                    link VARCHAR(30) NOT NULL UNIQUE,
+                    reg_date TIMESTAMP)"""
+                    cur.execute(query)
+                    for link in links:
+                            query = "INSERT IGNORE INTO `tor_url` (link) VALUES ('{0}')".format(link)
+                            cur.execute(query)
+                            #print(query)
+                    db.commit()
+
+            except (MySQLdb.Error, MySQLdb.Warning, TypeError, ValueError) as e:
+                    print(e)
```

Comment:
Why not raise error instead of printing and returning None?

Reply:
I wasn't considering raising an error for this... If you can't save to the database and raise an error the program will stop... If that is a requirement I can change it but in my opinion, a simple message is better

Reply:
It's not a requirement, I was just curious. It's fine as it is though :)

Reply:
@KingAkeem want me to raise an error if unable to write to the table??

Reply:
No, printing is fine. If we find a need to raise errors over using print statements then we will.